components/PageHeader.vue

```diff
@@ -15,39 +15,29 @@
       >
     </b-navbar-brand>
     <b-navbar-nav class="ml-auto w-100 col-md-5 col-lg-4 p-0">
-      <b-nav-form
-        class="justify-content-center justify-content-sm-end"
-        @submit.prevent="submitSearchForm"
-      >
-        <b-form-input
-          v-model="query"
-          size="sm"
-          type="text"
-          class="mr-2 w-75"
-          placeholder="What are you looking for?"
-          name="query"
-          data-qa="search box"
-        />
-        <b-button
-          size="sm"
-          type="submit"
-          data-qa="search button"
-        >
-          <img
-            src="../assets/img/magnifier.svg"
-            alt="Search"
-          >
-        </b-button>
-      </b-nav-form>
+      <SearchForm
+        v-model="query"
+        :is-loading="isLoading"
+        :in-header="inHeader"
```

Comment:
the in header 'argument' is a bit too specific perhaps. Since when it gets detected all it does is set the input size to small,  let's call the SearchForm component with that information here already.

Reply:
I have removed the inHeader completely, so there is no longer a prop for styling purposes. This way it is simple and functional for now and we can make design modifications later.